packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js

```diff
@@ -0,0 +1,55 @@
+const _ =  require('underscore');
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const { existsSync, readFileSync } = require('fs-extra');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing i18nkeys from json mock files',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      missingApiKeysLoginBundle: 'Missing i18n key from login.properties returned from the API. Add {{i18nKey}} to login.properties to prevent english leaks',
+      missingi18nKeyApiResponse: 'Missing i18n key in API mock {{file}}. Avoid using hard-coded strings returned and instead have the API return an i18n key and add that key to login.properties to prevent english leaks'
+    },
+  },
+  create(context) {
+    return {
+      'Program': function (node) {
+        const fileName = context.getFilename();
+          const jsonContent = JSON.parse(readFileSync(fileName, 'utf8'));
+          const messages = jsonContent.messages;
+          let loc = node.loc.start;
+          // check if i18n keys returned from API  are added to login.properties
+          if(messages && messages.value[0] && messages.value[0].i18n) {
```

Comment:
Since we're expecting `messages` and `messages.value` to be defined, let's invert this logic to `return` when it is not found to move our logic outside of the conditional:

```js
if (!messages || !messages.value || !messages.value.length) {
  return;
}

const { i18n } = messages.value[0];
const i18nKey = i18n.key;
...
```